packages/devtools_app/lib/src/inspector/flutter/inspector_tree_flutter.dart

```diff
@@ -83,18 +84,21 @@ class _InspectorTreeRowState extends State<_InspectorTreeRowWidget>
 
 class InspectorTreeControllerFlutter extends Object
     with InspectorTreeController, InspectorTreeFixedRowHeightController {
+  // Controller for controlling custom animation in the Tree
+  AnimationController animationController;
```

Comment:
remove this animationController as it doesn't seem to be needed and it is better to put the AnimationController int eh flutter widget classes not the controller classes.

Reply:
Yes, I agree, initially I wanted to do that but I was too lazy with the "prop drilling" (passing the object > 1 level down the widget tree).